lib/plugins/TaskRunner/Drupal.js

```diff
@@ -38,7 +38,7 @@ class Drupal extends LAMPApp {
   constructor(container, options) {
     super(container, options);
 
-    this.databaseName = constants.DRUPAL_DATABASE_NAME;
```

Comment:
These constants seemed pointless to me. They just make it harder to see what the defaults actually are.